exalearn-ui/src/app/services/table.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@angular/core';
```

Comment:
Please leave TODO to remove it in future

Reply:
Do you mean the whole service?

Reply:
Now here us only method with test data, so yes, just keave a comment TODO to remove test sata after backed integration